### DISCUSSION TITLE
How can I make it perfectly sure that footfall count occurs only when the detection cross the line

### DISCUSSION THREAD

**Discussion by xwhiz** [roboflow/supervision#519] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Hi,

I've been experimenting with the LineZone feature in the supervision module and noticed that it currently divides the entire frame into two sections, one for True and the other for False. However, I've observed that even when a detection doesn't touch the line, it still gets counted.

Is there a way to enhance this feature so that it only counts when detection is actually touching the line? I'd greatly appreciate any guidance on this matter.

Warm regards,

**Comment by a2082761** [comment]
Has this issue been resolved? I am also experiencing the same problem.

**Reply by rolson24** [reply]
What version of supervision are you using? I believe in 0.18.0 they fixed this by adding different anchor points. For example, if you only want to count when the bottom corners of a bounding box cross, you can use `(Position.BOTTOM_RIGHT, Position.BOTTOM_LEFT)` as your triggering anchors argument when creating the LineZone object.
